### PR TITLE
fix: cleanup `package-utils` package

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**

refactor

**Did you add tests for your changes?**
Migrated tests

**If relevant, did you update the documentation?**
NA

**Summary**
- This removes the package `package-utils` which was kinda redundant since we already had `utils` package

**Does this PR introduce a breaking change?**
Shouldn't 

**Other information**

### CODE REVIEW

**Review by rishabh3112 — COMMENTED** [review]
Left a suggestion

File: packages/webpack-cli/package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
     "@webpack-cli/info": "^1.0.1-alpha.4",
     "@webpack-cli/init": "^1.0.1-alpha.5",
     "@webpack-cli/serve": "^1.0.1-alpha.5",
+    "@webpack-cli/utils": "^1.0.1-alpha.5",
```
[the remark this answers is not in the document]
Comment (rishabh3112) [reply packages/webpack-cli/package.json:29]: Lets keep `@webpack-cli/utils` for utils which are not required by main CLI but other sub packages.

[the remark this answers is not in the document]
Comment (anshumanv) [reply packages/webpack-cli/package.json:29]: There are functions which are needed by other packages as well

[the remark this answers is not in the document]
Comment (rishabh3112) [reply packages/webpack-cli/package.json:29]: Yes, that's what I said [here](https://github.com/webpack/webpack-cli/pull/1822#discussion_r492807748).
If utils is used by main CLI (and also be used by other sub-packages) -> move to `webpack-cli`
If utils is not used by CLI and used by other sub packages -> move to `@webpack-cli/utils`

[the remark this answers is not in the document]
Comment (anshumanv) [reply packages/webpack-cli/package.json:29]: Let's see

[the remark this answers is not in the document]
Comment (anshumanv) [reply packages/webpack-cli/package.json:29]: will check after CI fix

**Review by evilebottnawi — APPROVED** [review]
Good job!

cc @webpack/cli-team

File: packages/webpack-cli/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (anshumanv) [reply packages/webpack-cli/package.json]: done

**Review by evilebottnawi — APPROVED** [review]

**Review by rishabh3112 — APPROVED** [review]

**Pull request merged**